### PR TITLE
Fix tests #383

### PR DESCRIPTION
**Pull request by Dougnkong**
#383 

 Fix the currently failing tests after changing the UI and some functionalities.

### CODE REVIEW

File: ios/OX Coi/Application/GeneratedPluginRegistrant.m [hunk 1]
```diff
@@ -0,0 +1,52 @@
+//
```
Comment (Boehrsi) [inline ios/OX Coi/Application/GeneratedPluginRegistrant.m]: Please remove this file from the commit.

File: ios/OX Coi/Application/GeneratedPluginRegistrant.h [hunk 2]
```diff
@@ -0,0 +1,14 @@
+//
```
Comment (Boehrsi) [inline ios/OX Coi/Application/GeneratedPluginRegistrant.h]: Please remove this file from the commit.

File: lib/src/background/background_manager.dart [hunk 3]
```diff
@@ -0,0 +1,114 @@
+/*
```
Comment (Boehrsi) [inline lib/src/background/background_manager.dart]: Please remove this file from the commit.

File: lib/src/background/background_event_state.dart [hunk 4]
```diff
@@ -0,0 +1,66 @@
+/*
```
Comment (Boehrsi) [inline lib/src/background/background_event_state.dart]: Please remove this file from the commit.

File: lib/src/background/background_bloc.dart [hunk 5]
```diff
@@ -0,0 +1,80 @@
+/*
```
Comment (Boehrsi) [inline lib/src/background/background_bloc.dart]: Please remove this file from the commit.

File: ios/exportOptions.plist [hunk 6]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Boehrsi) [inline ios/exportOptions.plist]: Please remove this file from the commit.

File: pubspec.lock [hunk 7]
```diff
@@ -140,7 +140,7 @@ packages:
       name: coverage
```
Comment (Boehrsi) [inline pubspec.lock]: Please remove this file from the commit.

File: tools/build.iosDevReleaseBuild.sh [hunk 8]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```
Comment (Boehrsi) [inline tools/build.iosDevReleaseBuild.sh]: Please remove this file from the commit.

File: tools/build.iosReleaseBuild.sh [hunk 9]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (Boehrsi) [inline tools/build.iosReleaseBuild.sh]: Please remove this file from the commit.

Comment (Dougnkong) [reply tools/build.iosReleaseBuild.sh]: Fixed: Files are remove.

**Comment by Boehrsi** [comment]
I just fixed two remaining files (images) which shouln't be in the commit and rebased everything + pushed it on this branch. It's now building again but some errors remain. Please have another look why some tests are failing / freeze. Don't forget to pull my changes.

File: test_driver/group_test.dart [hunk 10]
```diff
@@ -52,14 +52,19 @@ import 'setup/helper_methods.dart';
...
+
+  tearDownAll(() async {
+    await teardownDriver(driver);
+  });
 
   const testNameGroup = "TestGroup";
   const newNameTestGroup = "NewNameTestGroup";
   const keyMoreButton11 = "keyMoreButton_11";
-  const keyMoreButton10 = "keyMoreButton_10";
+  const keyMoreButton10 = "keyMoreButton_12";
```
Comment (florianhaar) [inline test_driver/group_test.dart]:
```suggestion
  const keyMoreButton12 = "keyMoreButton_12";
```

Comment (florianhaar) [reply test_driver/group_test.dart]: the naming should be like this

File: lib/src/utils/keyMapping.dart [hunk 11]
```diff
@@ -43,56 +43,56 @@
...
-const keyUserProfileSettingsAdaptiveIcon= "keyUserProfileSettingsAdaptiveIcon";
+const keyProfileHeaderAdaptiveIconButton = "keyProfileHeaderAdaptiveIconButton";
+const keyProfileHeaderText = "keyProfileHeaderText";
+
+const keyUserProfileSettingsAdaptiveIcon = "keyUserProfileSettingsAdaptiveIcon";
+const keyUserProfileFlagIconSource = "keyUserProfileFlagIconSource";
+const keyUserProfileAvatarIconSource = "keyUserProfileAvatarIconSource";
+const keyUserProfileQrIconSource = "keyUserProfileQrIconSource";
+const keyUserProfileChatIconSource = "keyUserProfileChatIconSource";
+const keyUserProfileSecurityIconSource = "keyUserProfileSecorityIconsource";
```
Comment (florianhaar) [inline lib/src/utils/keyMapping.dart]: >
```suggestion
const keyUserProfileSecurityIconSource = "keyUserProfileSecurityIconsource";
```
typo

File: lib/src/utils/keyMapping.dart [hunk 12]
```diff
@@ -43,56 +43,56 @@
...
+const keyUserProfileBugReportIconSource = "keyUserProfileBugReportIconSource";
 
 const keyRootIconChatTitleText = "Chat";
 const keyRootIconContactsTitleText = "Contacts";
 const keyRootIconProfileTitleText = "Profile";
 
 const keyUserSettingsCheckIconButton = "keyUserSettingsCheckIconButton";
-const keyUserSettingsUserSettingsUsernameLabel =
-    "key_UserSettingsUserSettingsUsernameLabel";
+const keyUserSettingsUserSettingsUsernameLabel = "key_UserSettingsUserSettingsUsernameLabel";
```
Comment (florianhaar) [inline lib/src/utils/keyMapping.dart]:
```suggestion
const keyUserSettingsUsernameLabel = "key_UserSettingsUsernameLabel";
```
is there a reason that "UserSettings" is twice in the name?

**Comment by Boehrsi** [comment]
Works now on the branch. Will test the rebased version tomorrow.

**Comment by Boehrsi** [comment]
Had to adjust some small things after the rebase. It seems to work now. Merged to develop. Fixed by https://github.com/open-xchange/ox-coi/commit/aa963fecdfd232777d2364a9054884bc1c76dde6

**Pull request closed without merging**